### ISSUE TITLE
Implementation of ModMenu

### ISSUE THREAD

**Issue by kmorales13** [OroArmor/Oro-Config#1]
Hello. I've implemented your library into my mod, and so far it works great.
I just have one small issue, and that is that the modMenu integration doesn't seem to work for me.

This is the error I get in the console:
`Mod provides a broken implementation of ModMenuApi`

My config class [https://pastebin.com/2hd5gr1d](https://pastebin.com/2hd5gr1d)

**Comment by OroArmor** [comment]
Could i see your modmenu entry point?

**Comment by kmorales13** [comment]
```
"entrypoints": {
    "main": ["net.sleeppunisher.SleepPunisher"],
    "modmenu": ["net.sleeppunisher.ModMenuIntegration"]
  },
```

```
package net.sleeppunisher;

import com.oroarmor.config.screen.ModMenuConfigScreen;

public class ModMenuIntegration extends ModMenuConfigScreen {
    public ModMenuIntegration() {
		super(SleepPunisher.CONFIG);
	}
}
```

**Comment by OroArmor** [comment]
and what version of modmenu are you using?

**Comment by kmorales13** [comment]
2.1.0

**Comment by OroArmor** [comment]
Modmenu, not oro config, sorry. Also apologies for the late response, I turned some of my github notifications down and it made things worse

**Comment by kmorales13** [comment]
ah my bad, its 1.16.8. no worries

**Comment by OroArmor** [comment]
Huh yeah everything should be fine. Do you have a repository I could look at/tinker with?

**Comment by kmorales13** [comment]
yup [here's my mod](https://github.com/kmorales13/SleepPunisher), it's missing the entrypoint btw

**Comment by ChocolateLoverRaj** [comment]
My mod also has a broken implementation.

**Comment by ChocolateLoverRaj** [comment]
> it's missing the entrypoint btw

@kmorales13 can you commit your latest changes and then tell us the branch (if it's on another branch).
